packages/web-components/src/utilities/type-ramp.ts

```diff
@@ -0,0 +1,12 @@
+/** @public */
+export enum StandardFontWeight {
```

Comment:
Enum's don't compress and I'd like to avoid them if possible. We've had reports from certain folks that in some scenarios they can't use them and a type is preferred. In this case, where it's not a type, we can use an object such as below:

```ts
export const StandardFontWeight = {
    Thin = 100,
    ExtraLight = 200,
    ...
} as const;